### DISCUSSION TITLE
No airdrop in Iran?

### DISCUSSION THREAD

**Discussion by ZProtons** [masterking32/MasterHamsterKombatBot#445] [Q&A]
Binance will support the distribution of the HMSTR airdrop claims for eligible users who choose to accept the distribution to their Binance account from the [HMSTR airdrop claim page](https://t.me/hamster_kombat_bot/) (the page will be available in around 24 hours from the publication of this announcement). If users claim the HMSTR airdrop but do not receive tokens in their Binance Spot Account no later than 1 hour before the listing time, please claim HMSTR on-chain or contact [HMSTR Telegram Bot](https://t.me/hamster_kombat_bot/).

Individuals residing in certain regions will not be able to participate in airdrop distribution including: Canada, Cuba, Crimea Region, Cyprus, Georgia, Iran, Ireland, Japan, Luxembourg, Netherlands, North Korea, Syria, United Kingdom, United States of America and its territories (American Samoa, Guam, Puerto Rico, the Northern Mariana Islands, the U.S. Virgin Islands), Uruguay, Uzbekistan and Venezuela. This list may be updated periodically at Binance’s discretion.

**Comment by Fy0urM** [comment]
i think it's Binance's limitations, just choose another option to get tokens
